opennms-doc/guide-install/src/asciidoc/text/minion/configure-minion.adoc

```diff
@@ -30,6 +30,15 @@ admin@minion()> scv:set opennms.http minion minion
 admin@minion()> scv:set opennms.broker minion minion
 ----
 
+Another way to configure credentials is to use the `scvcli` utility in your _Minion_ `bin` directory.
+
+.Configure the credentials by using the command line utility `scvcli`
+[source]
+----
+[root@localhost /usr/share/minion]# $ ./bin/scvcli set opennms.http minion minion
```

Comment:
Ok. You're right. It was the Debian-related stuff. I point the path to /opt/minion, right?